scripts/test-templates.sh

```diff
@@ -47,81 +52,77 @@ dotnet new stacks-cqrs -n Cqrs.ServiceBus -do Menu --cloudProvider Azure --cicdP
 dotnet new stacks-cqrs -n Cqrs.Dynamo -do Menu --cloudProvider AWS -db DynamoDb  
 dotnet new stacks-cqrs -n Cqrs.Sns -do Menu --cloudProvider AWS -e AwsSns
 
-
 # Generate the additional template
 dotnet new stacks-webapi -n NonCqrs -do Menu
-cd NonCqrs/src/simple-api/src/api
+cd $TEST_TEMPLATES_DIRECTORY/NonCqrs/src/simple-api/src/api
 dotnet new stacks-add-cqrs -n NonCqrs.Cqrs -do Menu
 
 # Test the Simple.Api project
-cd Simple.Api/src/simple-api/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Simple.Api/src/simple-api/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the Cqrs project
-cd Cqrs/src/cqrs/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Cqrs/src/cqrs/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the Cosmos.Worker project
-cd Cosmos.Worker/src/func-cosmosdb-worker/src/functions
+cd $TEST_TEMPLATES_DIRECTORY/Cosmos.Worker/src/func-cosmosdb-worker/src/functions
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the EventHub.Listener project
-cd EventHub.Listener/func-aeh-listener/src/functions
+cd $TEST_TEMPLATES_DIRECTORY/EventHub.Listener/func-aeh-listener/src/functions
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../..
 
 # Test the ServiceBus.Listener project
-cd ServiceBus.Listener/func-asb-listener/src/functions
+cd $TEST_TEMPLATES_DIRECTORY/ServiceBus.Listener/func-asb-listener/src/functions
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../..
 
 # Test the ServiceBus.Worker project
-cd ServiceBus.Worker/background-worker/src/worker
+cd $TEST_TEMPLATES_DIRECTORY/ServiceBus.Worker/background-worker/src/worker
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../..
 
 # Test the Cqrs.AllTheThings project
-cd Cqrs.AllTheThings/src/cqrs/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Cqrs.AllTheThings/src/cqrs/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the Cqrs.ServiceBus project
-cd Cqrs.ServiceBus/src/cqrs/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Cqrs.ServiceBus/src/cqrs/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the Cqrs.Dynamo project
-cd Cqrs.Dynamo/src/cqrs/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Cqrs.Dynamo/src/cqrs/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
 
 # Test the Cqrs.Sns project
-cd Cqrs.Sns/src/cqrs/src/api
+cd $TEST_TEMPLATES_DIRECTORY/Cqrs.Sns/src/cqrs/src/api
 dotnet restore
 dotnet build
 dotnet test
-cd ../../../../..
+
+# Test the NonCqrs project
```

Comment:
This was intentionally avoided as it doesn't build without manual intervention. Because it is a separate project generated programatically over the top of an existing project it has issues with referencing.